pkg/eventmonitor/consumers/process.go

```diff
@@ -23,17 +23,17 @@ type ProcessCallback = func(uint32)
 // ProcessConsumer represents a consumer of process exec/exit events that can be subscribed to
 // via callbacks
 type ProcessConsumer struct {
-	// id is the identification of this consumer in the event data stream
-	id string
-
-	// chanSize is the channel size that will be used to send events to this consumer from the event data stream
-	chanSize int
+	// options holds the options that were passed to NewProcessConsumer
+	options ProcessConsumerOptions
 
 	// execCallbacks holds all subscriptors to process exec events
 	execCallbacks callbackMap
 
 	// exitCallbacks holds all subscriptors to process exit events
 	exitCallbacks callbackMap
+
+	// forkCallbacks holds all subscriptors to process fork events
+	forkCallbacks callbackMap
```

Comment:
consider creating those only if the ProcessConsumer was created with the proper flag